packages/flutter_tools/test/integration.shard/android_plugin_compilesdkversion_mismatch_test.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
```

Comment:
@jmagman is it possible to migrate this to null-safe, or is there a dependency that is preventing this from happening?

Reply:
Yes, this file should be made null safe.
@camsim99 remove `// @dart = 2.8`, and update
```
Directory tempDir;
```
to
```
late Directory tempDir;
```